prep-html-syntax/index.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+  <h1 style="color: rgb(77, 159, 236);"> My favorite character is Obi-Wan Kenobi</h1>
+  <h2><img src="https://nerdbot.com/wp-content/uploads/2020/01/1r3fn3-e1579990017466.jpg" alt="Kenobi"></h2>
+  <p style="color: rgb(68, 181, 216);"> A great Jedi that always tried to do the right thing. </p>
```

Comment:
Hi Derek, be sure to divide your page into two specific sections. The `<head>` should open and close before the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```